ssl_certs/localhost.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Usually TLS certificates have expiration dates and some day they will expire. It would be good to describe somewhere steps to generate new certificates.

Reply:
I agree that steps to re-generate certificates should be described. I can share my old intructions how to generate testing certificates:

```shell
$ cat deploy/ssl_ca/README
# Gen CA key (password protected) & cert
openssl genrsa -aes256 -out ca.key 4096
chmod 400 ca.key
openssl req -new -x509 -sha256 -key ca.key -out ca.crt -days 365 

# Gen client key & CSR (can be performed on a client side)
openssl genrsa -out client-foo.key 4096
chmod 400 client-foo.key
openssl req -new -key client-foo.key -sha256 -out client-foo.csr

# Gen cert for a client
openssl x509 -req -sha256 -in client-foo.csr -CA ca.crt -CAkey ca.key -set_serial 1 -out client-foo.crt -days 365 
```

```shell
$ cat deploy/ssl_server/README 
# Gen server key (no password to using in nginx) & cert
openssl req -x509 -sha256 -nodes -newkey rsa:4096 -keyout server.key -out server.crt -days 365 
```

```
$ cat deploy/ssl_client/README 
# Gen client key, CSR, and cert for ${SERVICE_NAME}
openssl genrsa -out ${SERVICE_NAME}.key 4096
chmod 400 ${SERVICE_NAME}.key
openssl req -new -key ${SERVICE_NAME}.key -sha256 -out ${SERVICE_NAME}.csr
openssl x509 -req -sha256 -in ${SERVICE_NAME}.csr -CA ../ssl_ca/ca.crt -CAkey ../ssl_ca/ca.key -set_serial 1 -out ${SERVICE_NAME}.crt -days 365 
```

However I don't recommend to generate one-year certificates. It was my mistake. It is quite annoying to find the testing unexpectedly broken because of that.

Reply:
Thank you, added. See ssl_certs/generate.sh

Reply:
(it was renamed to `testdata/generate.sh`)